site/specs-temp/numbers.json

```diff
@@ -19703,6 +19703,425 @@
+          },
+          {
+            "$ref": "#/components/parameters/OrderIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The information has been successfully retrieved and displayed in the payload.",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/PortinLosingCarrierResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/uctrunks": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Get sip peer premise trunks",
+        "operationId": "GetSipPeerPremiseTrunks",
+        "summary": "Get SipPeer Premise Trunks",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          },
+          {
+            "$ref": "#/components/parameters/SiteIdPathParam"
+          },
+          {
+            "$ref": "#/components/parameters/SipPeerIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Information was successfully retrieved.",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/UcTrunkingPremiseTrunksResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/products/{productName}/tntotals": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Get Product Tn Totals",
+        "operationId": "GetProductTnTotals",
+        "summary": "Get Product Tn Totals",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          },
+          {
+            "name": "productName",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "Origination",
+                "Messaging",
+                "TollFree"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Information was successfully retrieved.",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProductTnHistoryResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/totalSeats": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the total number of seats on sip peer location.",
+        "operationId": "GetSipPeerTotalSeats",
+        "summary": "Get Sip Peer Total Number of Seats.",
```

Comment:
```suggestion
        "summary": "Get Sip Peer Total Number of Seats",
```

Reply:
I don't think you addressed this comment yet right?

Reply:
Sorry, Michael, thought I done with this :/